utils/nctl/sh/scenarios-upgrades/upgrade_scenario_05.sh

```diff
@@ -138,7 +138,7 @@ function _step_03()
     done
 
     log "... awaiting 2 eras + 1 block"
-    await_n_eras '2' 'true' '5.0' '2'
+    nctl-await-n-eras offset='2' sleep_interval='5.0' timeout='180' node_id='2'
```

Comment:
Personally, I'm not sure whether the `nctl` alias makes much sense. All `nctl-*` commands available from the shell are "useful" for CLI users to interact with the network. I can't imagine a case where one would use `nctl-await-n-eras` there. Not a blocker obviously.